src/frontend/src/flows/authenticate/index.ts

```diff
@@ -115,19 +109,22 @@ const pageContent = (
       <div class="l-stack">
         <ul class="c-list--flex">
           <li>
-            <a ${ref(registerButton)} id="registerButton" class="t-link"
+            <a @click=${register} id="registerButton" class="t-link"
               >Create Anchor</a
             >
           </li>
           <li>
-            <a @click="${onRecoverClick}" id="recoverButton" class="t-link"
+            <a
+              @click="${() => recoverAnchor(anchorInput.readUserNumber())}"
+              id="recoverButton"
+              class="t-link"
               >Lost Access?</a
             >
           </li>
+        </ul>
+        <ul class="c-list--flex">
           <li>
-            <a @click="${onManageClick}" class="t-link" id="manageButton"
-              >Manage Anchor</a
-            >
+            <a href="/" class="t-link" id="manageButton">Home</a>
```

Comment:
there is no state or store in memory that needs to be reseted on navigation right?

Just asking because we used to reload the window which woudl had for effect to reset potential stores, just in case though better asking

Reply:
but maybe current routing acts also as a reload anyway right

Reply:
Nope, nothing! At least for now